plugins/module_utils/redfish_utils.py

```diff
@@ -3394,3 +3394,74 @@ def get_hpe_fan_percent_min(self):
             fan_percent_min_config = hpe.get('FanPercentMinimum')
         result["fan_percent_min"] = fan_percent_min_config
         return result
+
+    def delete_all_volumes(self):
+        # Find the Storage resource from the requested ComputerSystem resource
+        response = self.get_request(self.root_uri + self.systems_uri)
+        if response['ret'] is False:
+            return response
+        data = response['data']
+        storage_uri = data.get('Storage', {}).get('@odata.id')
+        if storage_uri is None:
+            return {'ret': False, 'msg': 'Storage resource not found'}
+
+        # Get Storage Collection
+        response = self.get_request(self.root_uri + storage_uri)
+        if response['ret'] is False:
+            return response
+        data = response['data']
+
+        # Collect Storage Controllers
```

Comment:
Should this be "storage subsystems"?

Reply:
Agreed, changed to "Collect Storage Subsystems"

Reply:
Resolving this conversation. Thanks!